src/Model/DataPoint.php

```diff
@@ -1,14 +1,17 @@
 <?php
 
-namespace Sunnysideup\ImageClickSpots\Modeel;
+namespace Sunnysideup\ImageClickSpots\Model;
+
+use JonoM\ImageCoord\ImageCoordField;
+use SilverStripe\ORM\DataObject;
 
 class DataPoint extends DataObject
 {
+    private static $table_name = 'Sunnysideup_clickspots_DataPoint';
+
     private static $db = [
        'Title' => 'Varchar',
        'Description' => 'HTMLText',
-       'XCoordinate' => 'Float', //todo: check if this works or try Varchar
-       'YCoordinate' => 'Float',
```

Comment:
When I had these there, there was a warning about duplicated fields.
Maybe these _are_ needed as db fields and need to removed from the form only. I'm not sure, since it's not clear to me what this module is meant to do in the first place lol